api/v1beta1/flinkcluster_types.go

```diff
@@ -568,6 +574,22 @@ type FlinkClusterStatus struct {
 	// The status of savepoint progress
 	Savepoint *SavepointStatus `json:"savepoint,omitempty"`
 
+	// When the controller creates new ControllerRevision, it generates hash string from the FlinkCluster spec
+	// which is to be stored in ControllerRevision and uses it to compose the ControllerRevision name.
+	// Then the controller updates nextRevision to the ControllerRevision name.
+	// When update process is completed, the controller updates currentRevision as nextRevision.
+
+	// CurrentRevision indicates the version of FlinkCluster.
+	CurrentRevision string `json:"currentRevision,omitempty"`
```

Comment:
Need to add all the new fields in the CRD doc.

Reply:
Done.